### ISSUE TITLE
[BUG]: ALL JOBS APPLIED TO  Mid-Senior level EVEN AFTER SETTING IT TO FALSE

### ISSUE THREAD

**Issue by Ishannaik** [AIHawk-co/Auto_Jobs_Applier#494]
### Describe the bug

the jobs are being applied to 90% senior level even if it is set to false for some reason

### Steps to reproduce

Set med senior level to false

### Expected behavior

apply without senior level

### Actual behavior

applied to senior level

### Branch

main

### Python version

3.10.1

### LLM Used

ChatGPt

### Model used

Gpt-4o-mini

**Comment by V01D-NULL** [comment]
I'm seeing the same thing. It doesn't actually seem to be applying the experience filter in linkedin

**Comment by V01D-NULL** [comment]
Looks like there isn't a standard for the config file which came back to bite the developers of this project. I'm seeing snake and camel case being thrown around very loosely.

The docs demand we use `experienceLevel` in the yaml config but the code will only append the query parameter to the linkedin search that corresponds with experience level if it's written in snake case (`experience_level`) in the config yaml.

Tl;dr
The config validator expects "experience level" to be written in one way while the code expects it another way.
I copy pasted my experience level to satisfy both the camel and snake case convention and it worked fine.

**Comment by cjbbb** [comment]
Fxied, Thanks!
